### PR TITLE
Bring Drones Back Plz

### PR DESCRIPTION
**Pull request by rhoemer22**
### Intent of your Pull Request
Im trying to bring drones back.

Explain what the purpose of your PR is.
Dont know if i did this right, but my copy+paste skillz are unmatched!

Mention if your PR is related to an issue.
ThatLing is a douche

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -59,6 +59,15 @@
...
 	crate_type = /obj/structure/closet/crate/large
+	
+/datum/supply_pack/emergency/droneshells
+	name = "Drone Shell Crate"
+	desc = "The station's little helpers. Contains three Drone Shells."
+	cost = 1000
+	contains = list(/obj/item/drone_shell,
+					/obj/item/drone_shell,
+					/obj/item/drone_shell)
+crate_name = "drone shell crate"
```
Comment (ShadowDeath6) [inline code/modules/cargo/packs.dm]: not lined up right

Comment (rhoemer22) [reply code/modules/cargo/packs.dm]: done

**Review by ThatLing — CHANGES_REQUESTED** [review]
not modular

**Comment by rhoemer22** [comment]
I dont understand why i should modulize it. It appears that the first pull request to remove them, wasnt modulized neither.
Please elaborate

**Comment by ThatLing** [comment]
That's because the removal PR came from /tg/

**Comment by ThatLing** [comment]
It's been 14 days since the last update so I'm closing this. Feel free to reopen/make a new PR when you update this.

**Pull request closed without merging**